balancer/rls/cache.go

```diff
@@ -163,7 +163,8 @@ func (l *lru) getLeastRecentlyUsed() cacheKey {
 //
 // It is not safe for concurrent access.
 type dataCache struct {
-	maxSize     int64 // Maximum allowed size.
```

Comment:
Is this a formatting change alone? My eyes are failing me. If so, can you please revert this.

Reply:
Oh whoops reverted.